doc/configJson.md

```diff
@@ -0,0 +1,37 @@
+# Server Configuration
```

Comment:
To get this new doc file to show up in the docs list it needs to be added to https://github.com/PrairieLearn/PrairieLearn/blob/master/mkdocs.yml